src/utils/DynamicContent/DynamicContent.ts

```diff
@@ -3,6 +3,7 @@ import { DynamicProgressBarContent } from '@src/utils/DynamicContent/DynamicProg
 export interface DynamicContent {
 	currentDayName: string;
 	currentDate: string;
+	currentTime: () => string;
```

Comment:
Is there a reason why this needs to be method instead of a getter?

Reply:
When it's a getter js seems to cache the value, so it wasn't updating.